Makefile

```diff
@@ -0,0 +1,33 @@
+.POSIX:
+SHELL=/bin/sh
+BASE_PATH="/"
+
+.PHONY: ci-test
+ci-test: 
+	@echo "Install dependencies" 
+	npm ci
+	@echo "Check code formatting"
+	npm run check-format
+	@echo "Lint code"
+	npm run lint
+	@echo "Generate production build"
+	npm run export
+
+.PHONY: ci-deploy-prod
+ci-deploy-prod:
+	@echo "Generate production build"
+	npm run export -- --basepath=$(BASE_PATH)
+	if ! [ -x "$$(command -v aws)" ]; then\
```

Comment:
This should be installed already in GitHub Action runner environment: https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md

We should not automatically install things as part of deploy flow, since it may mess up user's local environment when manual deploying. Instead, we can either have a separate target for installing dependencies, or just prompt user to install it.